src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -25,15 +25,13 @@ col-sm-2
 {# Rows #}
 
 {% block form_row -%}
-{% spaceless %}
     <div class="form-group{% if (not compound or force_error|default(false)) and not valid %} has-error{% endif %}">
-        {{ form_label(form) }}
+        {{- form_label(form) -}}
         <div class="{{ block('form_group_class') }}">
-            {{ form_widget(form) }}
-            {{ form_errors(form) }}
+            {{- form_widget(form) -}}
+            {{- form_errors(form) -}}
         </div>
-    </div>
-{% endspaceless %}
+{##}</div>
```

Comment:
It was the only why I could think of to have a 'space control' between the two closing div tags with out having to use spaceless. I guess the last closing div can also be moved directly to the end of the previous one, but that will break the indents.